app/components/welcome/welcome.js

```diff
@@ -274,12 +274,11 @@ const dialogs = [
           <Anchor href='https://chrishartgerink.typeform.com/to/VNfDMq'>
             testing
           </Anchor>
-          . If you're interested in becoming a supporting member, get in touch
-          at{' '}
-          <Anchor href='mailto:community@libscie.org'>
-            community@libscie.org
-          </Anchor>
-          .
+          . Want to financially support us? Become a{' '}
+          <Anchor href='https://blog.libscie.org/p/6569c48c-7f53-4c63-ab68-e654a54abe96/'>
+            supporting member
+          </Anchor>{' '}
+          for €70 a year.
```

Comment:
```suggestion
          </Anchor>.
```

The price might update more often than this up

Reply:
Made a note in Notion so we don't forget to change this if the price changes.